src/components/DialogContent.tsx

```diff
@@ -20,6 +20,12 @@ import { PlainSubstitution } from "../types/translations";
 
 export interface DialogProps {
```

Comment:
while you're extending this object, please could you add a doc-comment which explains what it does?